configuration/docker_image_project_name.txt

```diff
@@ -0,0 +1 @@
+imaqal_covid19
```

Comment:
```suggestion
imaqal-covid19
```
for consistency with the other names.
I _think_ the naming convention for Docker is lower case and hyphen separated

Reply:
updated thanks